### DISCUSSION TITLE
retrainging the model myself

### DISCUSSION THREAD

**Discussion by mosescohen1** [openai/whisper#2000] [Q&A]
Hi everybody

I saw that the model card states that the model is less trained in languages other then english.

In my project, I have no use for english, I need other languages, so my question is: Is there a way that i can create training data myself, and train the model localy on my machine using this data, to try and improve the performance in the languages I need?
And if there is, how exactly would I do that?

Thanks a lot in advance

**Comment by phineas-pta** [comment]
https://huggingface.co/blog/fine-tune-whisper
